server/service/osquery_utils/queries_test.go

```diff
@@ -1903,6 +1903,22 @@ func TestDirectIngestHostCertificates(t *testing.T) {
 		"sha1":              "9c1e9c00d8120c1a9d96274d2a17c38ffa30fd31",
 	}
 
+	// row2 will not be ingeted because of the issue field containing an extra /
```

Comment:
As a follow up product/engineering question, we need to reassess our parsing assumptions based on a representative sample of certificates we know we want to support.